src/python/grpcio/grpc/__init__.py

```diff
@@ -649,6 +649,10 @@ def unary_unary(
 
     Args:
       method: The name of the RPC method.
+      request_serializer: Optional method for serializing the request
```

Comment:
When describing a behavior type, be clear about its parameter types and return types. In this case we should say something like "an optional behavior taking a request and returning the byte string that is the serialization of the request or None if the application uses byte strings for requests". But I'm having trouble expressing something reasonably short, clear to all programmers, and not insultingly obvious to most programmers. :-P